### PR TITLE
CRW-3137, updating with changes for building in cachito.

### PR DESCRIPTION
**Pull request by SDawley**
Updating with the changes that worked in my devel branch. Technically bootstrap.Dockerfile isn't needed anymore but I've left it since it might be useful to anyone who wants to build locally.

### CODE REVIEW

File: devspaces-imagepuller/get-sources.sh [hunk 1]
```diff
@@ -31,110 +31,17 @@ function log()
 	fi
 }
 
-if [[ ${PULL_ASSETS} -eq 1 ]]; then
```
Comment (nickboldt) [inline devspaces-imagepuller/get-sources.sh:34]: I guess on lines 12 and 16 you don't need PULL_ASSETS anymore. Should the `-p` flag exit 0 now?

**Comment by nickboldt** [comment]
If someone wants to build locally, can they use this?
```
docker build -t ${DOCKERIMAGE_NAME}:${DOCKERIMAGE_TAG} -f ./docker/Dockerfile .
```
(As seen on https://github.com/redhat-developer/devspaces-images/tree/devspaces-3-rhel-8/devspaces-imagepuller#manual  ) 

And if so, is bootstrap.Dockerfile therefore obsolete?

**Comment by SDawley** [comment]
I forgot the docker folder existed. bootstrap.Dockerfile is removed.

**Comment by nickboldt** [comment]
In sync.sh we're still creating the bootstrap.Dockerfile (lines 74-75)
```
# use upstream dockerfile as bootstrap one (to retrieve dependencies for offline build)
cp ${SOURCEDIR}/docker/Dockerfile ${TARGETDIR}/bootstrap.Dockerfile
```

**Review by nickboldt — CHANGES_REQUESTED** [review]
In sync.sh we're still creating the bootstrap.Dockerfile (lines 74-75)
```
# use upstream dockerfile as bootstrap one (to retrieve dependencies for offline build)
cp ${SOURCEDIR}/docker/Dockerfile ${TARGETDIR}/bootstrap.Dockerfile
```

Also if I compare traefik's get-sources.sh vs. this one, I see some other leftovers...

image

**Pull request merged**